javatests/arcs/core/storage/driver/DatabaseDriverTest.kt

```diff
@@ -348,18 +364,29 @@ class DatabaseDriverTest {
     assertThat(receiverVersion).isEqualTo(1)
   }
 
+  @Test
+  fun send_throws_ifUnsupportedDataType() = runBlockingTest {
+    val driver = buildDriver<Int>(database)
+
+    assertFailsWith(
+      UnsupportedOperationException::class,
+      "Unsupported type for DatabaseDriver: ${Int::class}"
```

Comment:
as we discussed in cl/349326855, this string parameter is not a message check, you need to check it separately

Reply:
Thanks!